src/main/java/evolvio/main/Main.java

```diff
@@ -0,0 +1,14 @@
+package evolvio.main;
+
+import javax.swing.JFrame;
+import javax.swing.WindowConstants;
+
+public class Main {
+    public static void main(String[] args) {
+        JFrame frame = new JFrame();
```

Comment:
Just for info: By adding code to display a window first will encurage others to get something rendered very soon. What can lead to overly being focused on the frontend part 😊. It's nothing negative but something to take note of.

Reply:
I'd like to have something fancy asap. No controls, but just moving circles on the screen. Then different people can work on different parts.

Reply:
That's true, will add a minimal rendering structure tonight as discussed